### PR TITLE
Add endertime

### PR DESCRIPTION
**Pull request by Akzentfrei**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/endertime/background.png
Comment (maaaathis) [inline minecraft_servers/endertime/background.png:1]: Compress this file

File: minecraft_servers/endertime/background@2x.png
Comment (maaaathis) [inline minecraft_servers/endertime/background@2x.png:1]: Compress this file

File: minecraft_servers/endertime/manifest.json
Comment (maaaathis) [inline minecraft_servers/endertime/manifest.json:1]: Correct your indentations

**Review by maaaathis — CHANGES_REQUESTED** [review]

**Comment by Akzentfrei** [comment]
compressed images

File: minecraft_servers/endertime/background.png
Comment (Akzentfrei) [reply minecraft_servers/endertime/background.png:1]: done

File: minecraft_servers/endertime/background@2x.png
Comment (Akzentfrei) [reply minecraft_servers/endertime/background@2x.png:1]: done

File: minecraft_servers/endertime/manifest.json
Comment (Akzentfrei) [reply minecraft_servers/endertime/manifest.json:1]: done

**Comment by DoJapHD** [comment]
Server (and website) is not yet released; reopen PR once server is released

**Pull request closed without merging**